images/slave/Dockerfile

```diff
@@ -54,11 +54,24 @@ RUN apt-get update && apt-get install -y apt-transport-https ca-certificates \
 RUN wget -O /usr/local/bin/docker-compose https://github.com/docker/compose/releases/download/1.8.1/docker-compose-Linux-x86_64 && \
     chmod +x /usr/local/bin/docker-compose
 
+# Sudo
+RUN apt-get update \
+    && apt-get install -y sudo \
+    && echo "jenkins ALL=NOPASSWD: /usr/bin/docker" >> /etc/sudoers \
```

Comment:
It seems so... but when you mount docker sockets from outside container then docker group may have different id then the one inside the container. In that case the only way to make it work is to use sudo (which always work).